exercises/4_6/index.html

```diff
@@ -0,0 +1,138 @@
+        margin-left: 0;
+        margin-right: auto;
+      }
+
+      .main-content .left-content .small-image {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        border-radius: 100%;
+      }
+
+      .main-content .right-content {
+        background-color: blue;
+        width: 60%;
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .main-content .description {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      // Adicione a tag h1 com o texto Exercício 4.5 - JavaScript DOM como filho da tag body;
+			let header1 = document.createElement("h1");
+			let headerTitle = document.createTextNode("Exercício 4.6 - JavaScript DOM");
+
+			header1.appendChild(headerTitle);
+			document.querySelector("body").appendChild(header1);
+
+			// Adicione a tag div com a classe main-content como filho da tag body;
+			let div1 = document.createElement("div");
+			div1.className = "main-content";
+			document.querySelector("body").appendChild(div1);
+
+			// Adicione a tag div com a classe center-content como filho da tag div criada no passo 2;
+			let div2 = document.createElement("div");
+			div2.className = "center-content";
+			document.querySelector("div").appendChild(div2);
+
+			// Adicione a tag p como filho do div criado no passo 3 e coloque algum texto;
+
+			let text = document.createElement("p");
+			text.innerHTML = "FILHO DA DIV!!!";
+			document.querySelector(".center-content").appendChild(text);
+
+			// Adicione a tag div com a classe left-content como filho da tag div criada no passo 2;
+
+			let leftContent = document.createElement("div");
+			leftContent.className = "left-content";
+			document.querySelector(".main-content").appendChild(leftContent);
+
+			// Adicione a tag div com a classe right-content como filho da tag div criada no passo 2;
+
+			let rightContent = document.createElement("div");
+			rightContent.className = "right-content";
+			document.querySelector(".main-content").appendChild(rightContent);
+
+			// Adicione uma imagem com src configurado para o valor https://picsum.photos/200 e classe small-image. Esse elemento deve ser filho do div criado no passo 5;
+
+			let image = document.createElement("img");
+			image.src = "https://picsum.photos/200";
+			image.className = "small-image";
+			document.querySelector(".left-content").appendChild(image);
+
+			// Adicione uma lista não ordenada com os valores de 1 a 10 por extenso, ou seja, um, dois, três, … como valores da lista. Essa lista deve ser filha do div criado no passo 6;
+			
+			let list = document.createElement("ul");
+			list.className = "unordered";
+			document.querySelector(".right-content").appendChild(list);
+
+			let stringArray = ["um", "dois", "três", "quatro", "cinco", "seis", "sete", "oito", "nove", "dez"];
+				for (index in stringArray) {
+					let listItem = document.createElement("li");
+					listItem.innerHTML += stringArray[index];
```

Comment:
Aqui nesse caso, se não me engano, você não precisaria ter colocado `+=`, daria pra ter colocado só `=` mesmo, porque não é um valor que está sendo somado a cada iteração. Como a variável é criada cada vez que o `for` roda, meio que o `stringArray[index]` é atribuído `listItem.innerHTML`, sem a necessidade de somar.

Reply:
Fala Pedro! Obrigado pelos comentários!
Cara, nesse caso eu escrevi a princípio sem a soma mesmo, mas ele sempre me retornava só o "um" por alguma razão mística... na tentativa e erro essa foi a única solução que achei no momento.